### PR TITLE
fix: test suite v0.3.10

### PR DESCRIPTION
**Pull request by ldhyen99**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
After pulling the test-untp repo and running Tier 2 tests with the current configuration, all tests are failing except for one DPP test. Reverting to a previous commit resolves the issue, indicating that changes made in commit [2e37154fa4f8b79c66a7a5bd7a4c2bc606545415](https://github.com/uncefact/tests-untp/commit/2e37154fa4f8b79c66a7a5bd7a4c2bc606545415#diff-82808d6236873bbd7f2c09b81cce1a20916f836dae45e2747f74b812eba39105L271) to the 0.3.10 and possibly 0.3.9 schemas/credentials are causing the failures.

This PR is implementing:
- Copy the traceabilityEvents, digitalFacilityRecord schema and data in Jargon for version v0.3.10.
- Update the schema and data for versions v0.3.10 of these schemas: aggregationEvent, associationEvent, objectEvent, transactionEvent, and transformationEvent.

The config file `credentials.json`
```json
{
  "credentials": [
    {
      "type": "aggregationEvent",
      "version": "v0.3.10",
      "dataPath": "credentials/AggregationEvent_instance-0.3.10.json",
      "url": ""
    },
    {
      "type": "associationEvent",
      "version": "v0.3.10",
      "dataPath": "credentials/AssociationEvent_instance-0.3.10.json",
      "url": ""
    },
    {
      "type": "conformityCredential",
      "version": "v0.3.10",
      "dataPath": "credentials/conformityCredential/DigitalConformityCredential_instance-0.3.10.json",
      "url": ""
    },
    {
      "type": "digitalFacilityRecord",
      "version": "v0.3.10",
      "dataPath": "credentials/digitalFacilityRecord/DigitalFacilityRecord_instance-0.3.10.json",
      "url": ""
    },
    {
      "type": "objectEvent",
      "version": "v0.3.10",
      "dataPath": "credentials/ObjectEvent_instance-0.3.10.json",
      "url": ""
    },
    {
      "type": "productPassport",
      "version": "v0.3.10",
      "dataPath": "credentials/productPassport/DigitalProductPassport_instance-0.3.10.json",
      "url": ""
    },
    {
      "type": "traceabilityEvents",
      "version": "v0.3.10",
      "dataPath": "credentials/traceabilityEvents/DigitalTraceabilityEvent_instance-0.3.10.json",
      "url": ""
    },
    {
      "type": "transactionEvent",
      "version": "v0.3.10",
      "dataPath": "credentials/TransactionEvent_instance-0.3.10.json",
      "url": ""
    },
    {
      "type": "transformationEvent",
      "version": "v0.3.10",
      "dataPath": "credentials/TransformationEvent_instance-0.3.10.json",
      "url": ""
    }
  ]
}
```

## Mobile & Desktop Screenshots/Recordings

`yarn run untp test`
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📜 README.md
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: packages/untp-test-suite/credentials-v0.3.10.json [hunk 1]
```diff
@@ -13,28 +13,38 @@
...
+      "version": "v0.3.10",
+      "dataPath": "credentials/DigitalTraceabilityEvent_instance-0.3.10.json"
+    },
+    {
+      "type": "transactionEvent",
       "version": "v0.3.10",
-      "dataPath": "credentials/DigitalProductPassport_instance-0.3.10.json",
-      "url": "https://jargon.sh/user/unece/DigitalProductPassport/v/0.3.10/artefacts/jsonSchemas/DigitalProductPassport.json?class=DigitalProductPassport"
+      "dataPath": "credentials/TransactionEvent_instance-0.3.10.json",
+      "url": ""
```
Comment (huynguyen-hl) [inline packages/untp-test-suite/credentials-v0.3.10.json]: I think we could remove the url property if it has no data here.

File: packages/untp-test-suite/README.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ The UNTP Test Suite is a tool that allows you to test your credentials against t
 - `objectEvent`: Contains the schema files for the object event (An object event describes an action on specific product(s) such as an inspection).
 - `transactionEvent`: Contains the schema files for the transaction event (A transaction event describes the exchange of product(s) between two actors such as sale of goods between seller and buyer).
 - `transformationEvent`: Contains the schema files for the transformation event (A transformation event describes a manufacturing process that consumes input product(s) to create new output product(s)).
+- `traceabilityEvents`: Contains the schema files for the traceability events. This schema defines a Digital Traceability Event as a Verifiable Credential (VC), enabling the tracking and documentation of events such as transactions, transformations, aggregations, and associations within a supply chain. It ensures standardized data exchange by encapsulating event details (e.g., time, location, and product identifiers) with context-driven semantics to support traceability and accountability.
```
Comment (huynguyen-hl) [inline packages/untp-test-suite/README.md:32]: I think we're missing information about the ObjectEvent in this description. Should we add a brief explanation to make it clearer?

File: packages/untp-test-suite/README.md [hunk 3]
```diff
@@ -29,6 +29,8 @@ The UNTP Test Suite is a tool that allows you to test your credentials against t
...
+- `digitalFacibilityRecord`: Contains the schema files for the digital facibility record. It holds performance and compliance information about a facility, such as sustainability metrics, standards, and regulatory conformity. It ensures structured data interoperability by referencing identifiers, classifications, and location details while supporting JSON-LD contexts for semantic meaning.
```
Comment (huynguyen-hl) [inline packages/untp-test-suite/README.md:33]: I'm a bit curious about the term `digitalFacibilityRecord` here. Did you mean `digitalFacilityRecord`?

**Review by huynguyen-hl — DISMISSED** [review]

**Review by huynguyen-hl — APPROVED** [review]

**Comment by ashleythedeveloper** [comment]
@ldhyen99, can we prefix the credential names with "digital" that don't have it to align with the established pattern? Same as ticket #139.

**Comment by ldhyen99** [comment]
> @ldhyen99, can we prefix the credential names with "digital" that don't have it to align with the established pattern? Same as ticket #139.

Yup, but when we merge [this PR](https://github.com/uncefact/tests-untp/pull/139), the `digital` will be applied

**Review by namhoang1604 — APPROVED** [review]

**Pull request merged**